storage/indexer/src/schema/mod.rs

```diff
@@ -7,12 +7,15 @@
 //! All schemas are `pub(crate)` so not shown in rustdoc, refer to the source code to see details.
 
 pub mod indexer_metadata;
+pub mod indexer_metadata_v2;
 pub mod table_info;
 
 use aptos_schemadb::ColumnFamilyName;
 
 pub const DEFAULT_COLUMN_FAMILY_NAME: ColumnFamilyName = "default";
+/// TODO(jill): to be deleted once INDEXER_METADATA_V2_CF_NAME is deployed
```

Comment:
note: you can't remove a column family from code unless you redo all the DB instances because RocksDB insists all existing CFs be mentioned in the open call. (but you can truncate the CF and rename the variable DEPRECATED_x)